jetty-core/jetty-fcgi/jetty-fcgi-server/src/main/java/org/eclipse/jetty/fcgi/server/internal/ServerFCGIConnection.java

```diff
@@ -178,8 +180,12 @@ public void onFillable()
                     LOG.debug("Read {} bytes from {} {}", read, getEndPoint(), this);
                 if (read > 0)
                 {
+                    // The networkBuffer cannot be released immediately after parse()
+                    // even if the buffer has been fully consumed because releaseInputBuffer()
+                    // must be called as the last release for it to be able to null out the
+                    // networkBuffer field exactly when the latter isn't used anymore.
                     if (parse(networkBuffer.getByteBuffer()))
-                        return;
+                        break;
```

Comment:
Is there any difference between `break` and `return` here?   or is this just a style thing?

Reply:
It's for style: there isn't any difference between `break` and `return` here. I just aligned this code with what the other branches were doing.

But If you prefer `return`, I'll change them all to use `return` instead of `break`.